cluster/saltbase/salt/top.sls

```diff
@@ -40,6 +40,9 @@ base:
     - kube-addons
 {% if grains['cloud'] is defined and grains['cloud'] == 'azure' %}
     - openvpn
+{% else %}
+    - docker
+    - sdn
```

Comment:
I believe this pull broke GCE.

I think this ended up enabling docker and sdn on *everything*, didn't it?